src/System.Runtime.Serialization.Xml/src/System.Runtime.Serialization.Xml.csproj

```diff
@@ -6,22 +6,24 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
   </PropertyGroup>
   <!-- Help VS understand available configurations -->
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='net463-Windows_NT-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='net463-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netfx-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netfx-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netcoreapp-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netcoreapp-Release|AnyCPU'" />
   <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
     <ProjectReference Include="$(SourceDir)System.Private.DataContractSerialization\src\System.Private.DataContractSerialization.csproj" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net463'">
-    <TargetingPackReference Include="System" />
-    <TargetingPackReference Include="System.Runtime.Serialization" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
+    <Reference Include="mscorlib" />
+    <Reference Include="System" />
+    <Reference Include="System.Runtime.Serialization" />
+    <ProjectReference Include="$(SourceDir)System.Runtime.Serialization.Primitives\src\System.Runtime.Serialization.Primitives.csproj" />
```

Comment:
Because we need the implementation assembly in order to unify types.  Regular reference would get the ref folder version which doesn't have the type forwards.  Runtime reference would be a race condition.  Project reference is same binary as runtime reference without the race condition.

There is a generic problem I was dealing with here and in other projects was what to do when we need to build a netfx facade for an assembly that is in NETStandard.Library.  For these we don't have a netstandard contract build.  I'm creating a netfx build but its kinda busted since we end up with a mix of facades and reference assemblies in the ref folder for netfx.  We have to have a reference assembly in order to produce the facade from source, while at the same time all the files from NETStandard.Library package in ref folder are facades.